Protocols/IEC61850/FlexOVL_540_171101_2/EventNotification.md

```diff
@@ -0,0 +1,55 @@
+## EventNotification messages
+
+### Description
+
+Light sensor device can send buffered report containing the current state of the digital inputs. The buffered report will be sent on data change.
+OSGP will interpret the buffered report and save event information contained in the reports.
```

Comment:
Provide some information about the content of an event, which is created in OSGP (data to the platform triggered by the event).